install/installer/pkg/config/v1/experimental/experimental.go

```diff
@@ -342,6 +351,7 @@ type IDEMetricsConfig struct {
 // @deprecated
 type VSXProxyConfig struct {
 	ServiceAnnotations map[string]string `json:"serviceAnnotations"`
+	ExtraEnvVars       map[string]string `json:"extraEnvVars"`
```

Comment:
where is it used?

Reply:
woops, that is from an old try before which I realized that config was deprecated